docker/test/integration/minifi/processors/FetchS3Object.py

```diff
@@ -20,4 +20,4 @@ def __init__(self,
                 'Proxy Username': proxy_username,
                 'Proxy Password': proxy_password,
             },
-            auto_terminate=['success'])
+            auto_terminate=['failure'])
```

Comment:
The FetchS3 tests could fail at the beginning when the data is not yet available on the test server. As there was no failure relationship for the processor it would get penalized for 30 seconds, and unnecessarily make the test run longer.

Reply:
I think we should keep `success`, too, as all the other S3 processors auto-terminate `success`, so this one behaving differently might surprise someone in the future.
```suggestion
            auto_terminate=['success', 'failure'])
```

Reply:
Added in 357be8ba107d5707546a5b053601b59c954b8477